operators/test/e2e/cmd/run/run.go

```diff
@@ -422,6 +432,15 @@ func (h *helper) renderTemplate(templatePath string, param interface{}) (string,
 	return outFilePath, nil
 }
 
+func (h *helper) deleteResources(file string) func() {
+	return func() {
+		log.Info("Deleting resources", "file", file)
+		if err := h.kubectl("delete", "--all", "--wait", "-f", file); err != nil {
```

Comment:
What is the purpose of `--all` when we use `-f file`?

Reply:
My understanding is that it should trigger the deletion of uninitialized resources as well. I am not sure whether the `-f` flag overrides that behaviour though.

Reply:
From my experience, `-f` seems to delete uninitialized resources but in the doubt I'm fine to keep `--all`.